### ISSUE TITLE
v3.1.3-beta.7 Label {"default":""} leer -> Geräte-Label wird trotzdem angezeigt, anstatt nichts

### ISSUE THREAD

**Issue by mcuiobroker** [Zefau/ioBroker.jarvis#2193]
https://forum.iobroker.net/topic/49776/jarvis-v3-0-0-just-another-remarkable-vis/3645

Nur mit
``` 
{"default":" "}
```
wird das Label in der StateList nicht angezeigt.

**Comment by Zefau** [comment]
Bei der Einstellung greift das Geräte Label?

**Comment by mcuiobroker** [comment]
Ja

![image](https://user-images.githubusercontent.com/84818934/214581503-dadd93d7-4ae2-498d-aca0-7f0c5f0c7b53.png)
![image](https://user-images.githubusercontent.com/84818934/214581557-ac4edf44-71df-4a6f-9c9a-360a7ac89fa4.png)

mit " "
![image](https://user-images.githubusercontent.com/84818934/214581662-a6577aec-3c82-4069-a3c1-91cec42d8ce0.png)
![image](https://user-images.githubusercontent.com/84818934/214581723-5b764387-c56f-4983-ab51-df0475cabee5.png)

**Comment by Zefau** [comment]
Dann passt es doch? :-) 

Der Fallback auf den Geräte Namen ist richtig, wenn es kein State Label gibt. `""` ist leer, daher der Fallback. Wenn `" "` funktioniert passt es ja. Eigl hätte ich gedacht, dass auch dann der Fallback zieht.

**Comment by Zefau** [comment]
@mcuiobroker ist das mit v3.2 noch aktuell?

**Comment by mcuiobroker** [comment]
Jo, Label wird noch immer aus dem Geräte-Name genommen, obwohl im Label `{"default":""}`
